addons/website/views/snippets/snippets.xml

```diff
@@ -559,13 +559,8 @@
             <we-button title="Contact"
                        data-name="header_contact_opt"
                        data-customize-website-views="website.template_header_contact"
-                       data-customize-website-variable="'Contact'"
+                       data-customize-website-variable="'contact'"
```

Comment:
Is this a bug you are fixing? Should not it be done in stable?

Reply:
moved to PR: https://github.com/odoo/odoo/pull/72933